2104SumofSubArrayRanges.py

```
@@ -0,0 +1,116 @@
+            for j in range(i,n):
+                results.append(nums[i:j+1])
+        for result in results:
+            min_value=max_value=result[0]
+            for i in range(1,len(result)):
+                if min_value>result[i]:
+                    min_value=result[i]
+                if max_value<result[i]:
+                    max_value=result[i]
+            total+=max_value-min_value
+        return total
+class  Solution:
```

Ruff (0.12.2)

Redefinition of unused `Solution` from line 47

(F811)